### PR TITLE
Update com.ibm.websphere.appserver.jwt-1.0.properties

### PR DESCRIPTION
**Pull request by ManasiGandhi**

### CODE REVIEW

**Review by dmuelle — APPROVED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/resources/l10n/com.ibm.websphere.appserver.jwt-1.0.properties [hunk 1]
```diff
@@ -14,4 +14,4 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature allows runtime to create JWT tokens.
+description=This feature allows the runtime to create JWTs.
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/resources/l10n/com.ibm.websphere.appserver.jwt-1.0.properties]: acronym isn't defined yet on the page so should be

This feature allows the runtime to create JSON web tokens (JWT)

**Review by dmuelle — CHANGES_REQUESTED** [review]
The acronym isnt defined yet on the page so the description should not use it

File: dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/resources/l10n/com.ibm.websphere.appserver.jwt-1.0.properties (hunk 1 above)
Comment (ManasiGandhi) [reply dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/resources/l10n/com.ibm.websphere.appserver.jwt-1.0.properties]: Updated

**Review by dmuelle — APPROVED** [review]
LGTM

**Review by dmuelle — APPROVED** [review]

**Review by cbridgha — APPROVED** [review]
LGTM

**Comment by dmuelle** [comment]
#libby

**Pull request merged**